### DISCUSSION TITLE
Is part of the intent to have historical WZ data?

### DISCUSSION THREAD

**Discussion by lynnerandolph** [usdot-jpo-ode/jpo-wzdx#101] [Q&A]
Texas Transportation Institute had asked whether they should implement a feed to provide the work zone information for I-35 in the Waco/Temple area of Texas. The stretch of roadway has had continuous work zones with lane closures for several years (moving, of course). Is part of the intent of this feed to gather some historical data such as this? 
Otherwise, current work zone information in that area is already part of the overall TxDOT feed. We didn't want them to go through this effort if that historical data was not wanted/needed.

**Comment by arielsgold** [comment]
@lynnerandolph Just jumping in here from my narrow perspective -- My program at USDOT is accumulating an archive by ingesting WZDx feeds (including TxDOT's, as you know) and offering this to the public for some period of time for the research community (within and outside USDOT) to experiment with. This will help us understand potential value in the archival data. We do not need historical data for this purpose. Interested in hearing from others who may have a different perspective.

**Comment by RHolt-RIDOT** [comment]
@lynnerandolph IOOs can certainly benefit from keeping an archive of data RE: where lane closures were in place, especially if they have other incident, crash, or mobility-related data available for analysis against/with such.  I think decision RE: whether and what specific work zone data to archive should be dependent on what resources and plans the IOO may actually have (if any) - if no capability to do any actual analysis, it may not be cost-effective (but then again, data storage seems to be getting cheaper and cheaper).  Forgive me if you already know all this - my small DOT is just getting geared up and don't have any more specific info to share yet sorry...
